<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPropertyEditorForm</class>
 <widget class="QWidget" name="ObjectPropertyEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Свойства объекта</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="shortName_lineEdit">
        <property name="placeholderText">
         <string>Отображаемое имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_lineEdit">
        <property name="placeholderText">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="description_plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="property_tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Отображение</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Что-то ещё</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="accept_pushButton">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_pushButton">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
